### PR TITLE
Upgrade for Spree >= 3.7

### PR DESCRIPTION
**Pull request by przemosk**
update travis configuration

Testing against Spree 3.7, 4.0, 4.1, 4.2

### CODE REVIEW

**Comment by przemosk** [comment]
@damianlegawiec @szymoniwacz feel free to review this PR

File: Appraisals [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
 end
 
-appraise 'spree-3-3' do
-  gem 'spree', '~> 3.3.0'
-  gem 'spree_auth_devise', '~> 3.3.0'
+appraise 'spree-4-2' do
+  gem 'spree', '~> 4.2.0.beta'
+  gem 'spree_auth_devise', '~> 4.2.0'
   gem 'rails-controller-testing'
 end
```
Comment (szymoniwacz) [inline Appraisals:12]: Please add spree-3-7 to Appraisals

Comment (przemosk) [reply Appraisals:12]: good point 😇

Comment (przemosk) [reply Appraisals:19]: fixed

**Review by szymoniwacz — APPROVED** [review]
LGTM :)

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,35 +1,10 @@
...
-    expectations.syntax = :expect
-  end
-
-  config.include Devise::TestHelpers, type: :controller
-end
-
-Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].each { |file| require file }
+# Requires supporting ruby files with custom matchers and macros, etc,
+# in spec/support/ and its subdirectories.
+Dir[File.join(File.dirname(__FILE__), 'support/**/*.rb')].sort.each { |f| require f }
```
Comment (przemosk) [inline spec/spec_helper.rb:10]: auto reject, missing  empty line

Comment (szymoniwacz) [reply spec/spec_helper.rb]: 👌

**Review by damianlegawiec — APPROVED** [review]

**Pull request merged**